lib/Doctrine/Common/Persistence/Mapping/Driver/SymfonyFileLocator.php

```diff
@@ -172,9 +181,21 @@ public function getAllClassNames($globalBasename = null)
 
                     // NOTE: All files found here means classes are not transient!
                     if (isset($this->prefixes[$path])) {
-                        $classes[] = $this->prefixes[$path].'\\'.str_replace('.', '\\', $fileName);
+
+                        // Calculate subspace as a relative path from basepath to file path
+                        $subspace = substr(realpath($file->getPath()), strlen(realpath($path)));
```

Comment:
I refer it as class nested namspace relative to given namespace prefix. I really don't know how to refer a part of namespace chain properly in english